### ISSUE TITLE
Error when CHUNKINI not correctly defined

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#234]
In GitLab by @ jlope2 on Jan 25, 2017, 11:50

It seems when ```CHUNKINI``` variable is defined but without a value assigned, then it crashes.

```
fmassonn@ bscesautosubmit01: /esnas/autosubmit/a0f1/conf >> autosubmit -v
3.7.7

fmassonn@ bscesautosubmit01: /esnas/autosubmit/a0f1/conf >> autosubmit create a0f1
Preparing .lock file to avoid multiple instances with same expid.

Checking configuration files...
autosubmit_a0f1.conf OK
platforms_a0f1.conf OK
jobs_a0f1.conf OK
expdef_a0f1.conf  OK
Configuration files OK

Cloning develop-nemo3.6-standalone http://earth.bsc.es/gitlab/es/auto-nemo.git into /esnas/autosubmit/a0f1/proj
Cloning into 'model'...
Username for 'http://earth.bsc.es': fmassonnet
Password for 'http://fmassonnet@ earth.bsc.es':
remote: Counting objects: 931, done.
remote: Compressing objects: 100% (58/58), done.
remote: Total 931 (delta 26), reused 0 (delta 0)
Receiving objects: 100% (931/931), 219.98 KiB | 0 bytes/s, done.
Resolving deltas: 100% (529/529), done.
Checking connectivity... done.
Cloning into 'ocean_diagnostics'...
remote: Counting objects: 2898, done.
remote: Compressing objects: 100% (2467/2467), done.
remote: Total 2898 (delta 2105), reused 549 (delta 390)
Receiving objects: 100% (2898/2898), 2.86 MiB | 1.96 MiB/s, done.
Resolving deltas: 100% (2105/2105), done.
Checking connectivity... done.
Cloning into 'plugins'...
remote: Counting objects: 15528, done.
remote: Compressing objects: 100% (18/18), done.
remote: Total 15528 (delta 5), reused 0 (delta 0)
Receiving objects: 100% (15528/15528), 22.61 MiB | 1.68 MiB/s, done.
Resolving deltas: 100% (5179/5179), done.
Checking connectivity... done.
Cloning into 'sources-nemo'...
Username for 'https://earth.bsc.es': fmassonnet
Password for 'https://fmassonnet@ earth.bsc.es':
remote: Counting objects: 47922, done.
remote: Compressing objects: 100% (8/8), done.
remote: Total 47922 (delta 0), reused 0 (delta 0)
Receiving objects: 100% (47922/47922), 816.29 MiB | 6.04 MiB/s, done.
Resolving deltas: 100% (37864/37864), done.
Checking connectivity... done.
Cloning into 'sources-xios'...
remote: Counting objects: 901, done.
remote: Compressing objects: 100% (515/515), done.
remote: Total 901 (delta 396), reused 882 (delta 380)
Receiving objects: 100% (901/901), 4.52 MiB | 4.74 MiB/s, done.
Resolving deltas: 100% (396/396), done.
Checking connectivity... done.
Cloning into 'specs2cmor'...
remote: Counting objects: 3585, done.
remote: Compressing objects: 100% (43/43), done.
remote: Total 3585 (delta 19), reused 0 (delta 0)
Receiving objects: 100% (3585/3585), 67.98 MiB | 5.84 MiB/s, done.
Resolving deltas: 100% (2568/2568), done.
Checking connectivity... done.
Loading parameters...
[CRITICAL] Unhandled exception on Autosubmit: invalid literal for int() with base 10: ''
Traceback (most recent call last):
  File "/shared/earth/software/autosubmit/3.7.7-foss-2015a-Python-2.7.9/lib/python2.7/site-packages/autosubmit-3.7.7-py2.7.egg/autosubmit/autosubmit.py", line 342, in parse_args
    return Autosubmit.create(args.expid, args.noplot, args.hide, args.output)
  File "/shared/earth/software/autosubmit/3.7.7-foss-2015a-Python-2.7.9/lib/python2.7/site-packages/autosubmit-3.7.7-py2.7.egg/autosubmit/autosubmit.py", line 1676, in create
    chunk_ini = as_conf.get_chunk_ini()
  File "/shared/earth/software/autosubmit/3.7.7-foss-2015a-Python-2.7.9/lib/python2.7/site-packages/autosubmit-3.7.7-py2.7.egg/autosubmit/config/config_common.py", line 663, in get_chunk_ini
    return int(self.get_option(self._exp_parser, 'experiment', 'CHUNKINI', 1))
ValueError: invalid literal for int() with base 10: ''
```

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Jan 25, 2017, 11:53

Mentioned in commit 5d7656a1d8f368829fe74ae3de8703b9b406e4c8

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Jan 25, 2017, 12:05

It's now fixed and the fix will be included in the next version.

However, in case you're using ```Autosubmit 3.8.0``` or an older version, try removing the line ```CHUNKINI``` of the ```expdef_XXXX.conf``` file, or set it equals to ```1```.

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Jan 25, 2017, 12:25

If you're suffering this problem, please point it out here.

If there are a lot of people suffering this problem, we will publish a new version ```3.7.8``` with this bug fixed.

Thanks!

**Comment by kinow** [comment]
In GitLab by @mcastril on Feb 7, 2017, 15:58

I also experimented these problem using the default version. I will point out in the wiki that people should fill or remove this argument.

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Feb 7, 2017, 16:08

Thanks @mcastril :)

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Feb 27, 2017, 15:18

Mentioned in commit 255b7abd0d4c75e00bb07861ada5b2dc81fe49d5

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Feb 27, 2017, 15:29

Mentioned in commit 629c695698ef645a860d29f3de53895e8759324b

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Feb 27, 2017, 18:17

Fixed! 

Available since Autosubmit v3.7.8.
